test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectHandlerTests.cs

```diff
@@ -422,14 +417,31 @@ private static void SecurityTokenValidatedSkippedOptions(OpenIdConnectAuthentica
                 };
         }
 
-        #endregion
+        private static void StateNullOptions(OpenIdConnectAuthenticationOptions options)
```

Comment:
All three of the methods here have the same implementation - is that deliberate?